lib/apiservers/engine/backends/container.go

```diff
@@ -264,7 +273,39 @@ func (c *Container) ContainerCreate(config types.ContainerCreateConfig) (types.C
 
 		h = addContRes.Payload
 	}
+	//Volume Attachment Section
+	msg, name, time := trace.Begin("Container.ContainerCreate - VolumeSection")
```

Comment:
If you want to use trace here instead of log.Debug then this should really be a separate helper function or wrapped in an inline function (with comment as to why) and use defer for trace.End. At the moment we have error paths that would result in no matching End output.
```
func() {
  defer trace.End(trace.Begin("Container.Create - volumes"))
  // code goes here
}()
```